maven-surefire-common/src/main/java/org/apache/maven/plugin/surefire/StartupReportConfiguration.java

```diff
@@ -76,27 +80,35 @@
 
     private final String xsdSchemaLocation;
 
-    private final Map<String, Deque<WrappedReportEntry>> testClassMethodRunHistory
-        = new ConcurrentHashMap<>();
+    private final Map<String, Deque<WrappedReportEntry>> testClassMethodRunHistory = new ConcurrentHashMap<>();
 
     private final Charset encoding;
 
-    private boolean isForkMode;
+    private final boolean isForkMode;
+
+    private final
+    StatelessReporter<WrappedReportEntry, TestSetStats, DefaultStatelessReportMojoConfiguration> xmlReporter;
```

Comment:
nit: remove new line after "final" ?

Reply:
I think here we can use shorter line `StatelessReporter`.